cmake/macros/testWrapper.py

```diff
@@ -313,4 +386,17 @@ def _runCommand(raw_command, stdout_redir, stderr_redir, env,
                                  '(DIFF_COMPARE).'.format(diff))
                 sys.exit(1)
 
+    if args.image_diff_compare:
+        converted = vars(args)
+        params = {key: converted[key] for key in
+                  ('warn', 'warnpercent', 'hardwarn',
+                   'fail', 'failpercent', 'hardfail', 'perceptual')
+                  if key in converted}
+
+        for image in args.image_diff_compare:
+            if not _imageDiff(image, args.baseline_dir, args.verbose, **params):
```

Comment:
This call to `_imageDiff` needs to pass along the `env` map that's set up above and pass it along to the call to `subprocess.call` so that the PATH changes via `--post-path` are respected. See the calls to `_runCommand` for examples.